### ISSUE TITLE
missing dataset

### ISSUE THREAD

**Issue by Neuronys** [AMontgomerie/question_generator#1]
Hi @iarfmoose 
Thanks for sharing this T5 example on question generation task.
The dataset you used for training is not included in the repo

`train_set = QGDataset(os.path.join(DIR, 'question_generator/datasets/qg_train.csv'))
train_loader = DataLoader(train_set, batch_size=BATCH_SIZE, shuffle=True)
valid_set = QGDataset(os.path.join(DIR, 'question_generator/datasets/qg_valid.csv')) 
valid_loader = DataLoader(valid_set, batch_size=BATCH_SIZE, shuffle=False)`

Would you be so kind to provide them ?
Thanks in advance
Philippe

**Comment by AMontgomerie** [comment]
Hi Philippe,

The datasets are too large to upload to the repo but [here's a google drive download link](https://drive.google.com/drive/folders/17ruVE8RSR6WDC3t3zIUY_1g18Xe5hdwb?usp=sharing)

**Comment by Neuronys** [comment]
Thanks a lot @iarfmoose 
Have you found this repo already: https://github.com/patil-suraj/question_generation ?
Cheers

**Comment by AMontgomerie** [comment]
No problem!

Yeah I came across that one after uploading my project. It seems to be a lot more in-depth than mine!
